src/components/dialogs/commons/modificationDialog.js

```diff
@@ -46,9 +46,24 @@ const ModificationDialog = ({
         closeAndClear(data, 'validateButtonClick');
     };
 
-    const handleValidationError = (errors) =>
+    const handleValidationError = (errors) => {
         onValidationError && onValidationError(errors);
 
+        // When scrolling to the field with focus, you can end up with the label not completely displayed
+        // We ensure that field with focus is displayed in the middle of the dialog
+        // Delay focusing to ensure it happens after the validation. without timout, document.activeElement will return the validation button.
+        setTimeout(() => {
+            const focusedElement = document.activeElement;
```

Comment:
you should use a useRef instead of document

Reply:
I agree, it's better to use useRef to pick a specific element and avoid problems that document.activeElement can sometimes cause